registry/repositories.go

```diff
@@ -4,7 +4,7 @@ type repositoriesResponse struct {
 	Repositories []string `json:"repositories"`
 }
 
-func (registry *Registry) Repositories() (repos []string, err error) {
```

Comment:
I think this conflicts with your other PR regarding compound return values. Which one are you preferring, right now?